pcsx2/Config.h

```diff
@@ -1143,6 +1143,8 @@ struct Pcsx2Config
 		BackupSavestate : 1,
 		SavestateZstdCompression : 1,
 		McdFolderAutoManage : 1,
+		ShowCheatsForAllCRCs : 1,
+		ShowPatchesForAllCRCs : 1,
```

Comment:
Doesn't need to be in the config struct, since it's UI-only.

Reply:
I configured the toggle setting with `SettingWidgetBinder` so that when the settings window is closed and re-opened it will retain it's value instead of always resetting to true.

Since `SettingWidgetBinder` puts the value as a GameSettings `.ini` setting, I added it to the `PCSX2Config`/`Config` so that when using the "Clear Settings" button it will clear this setting as well.

Is that not needed, or is there a different way to go about it?
I was under the impression setting names needed to be in `Config.h` in order to get picked up by the config clearing system.

Reply:
Only core settings should be in the `Pcsx2Config` struct.

The whole `ClearConfiguration()` ordeal was only a thing to prevent patch/cheat/input profile settings from getting wiped out when config gets reset. If we don't want to preserve the patch/cheat/input profile/UI settings, then it'd be better to scrap that function entirely, back up the handful of settings that you want preserved explicitly, then just blow the entire ini away.

Reply:
Fair enough, well it's certainly in no way critical to have this incorporated into 'Clear Settings' , it's only a click away from it's default setting. I'll remove it from the config classes and push the changes up.

Reply:
Forgot to mention, it'll also create a field in the global config, which isn't really desirable.